docs/source/technical_tutorials/access/aws_access.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    username = "harshinigirish"
</pre><p>remove this hard coded line, and use the code generated username from above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>